libqtile/widget/cmus.py

```diff
@@ -45,6 +46,12 @@ def __init__(self, **config):
         self.status = ""
         self.local = None
 
+        mouse_callbacks = {
+            'Button1': self.play,
+            'Button4': partial(subprocess.Popen, ['cmus-remote', '-n']),
+            'Button5': partial(subprocess.Popen, ['cmus-remote', '-r']),
+        }
```

Comment:
This seems like an unassigned local that never goes anywhere? `self.mouse_callbacks` maybe?